documentation/modules/security/proc-installing-certs-per-listener.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='proc-installing-certs-per-listener-{context}']
+= Installing your own CA certificates for listeners
+
+You can install your own certificates and public keys for Kafka brokers on a per-listener basis. These certificates can be signed by an external Certificate Authority (CA), such as your company's internal CA or a public CA, rather than the cluster or clients CAs (which are internal to {ProductName}). Installing per-listener certificates is supported for two types of listeners:
+
+* TLS listeners for inter-cluster communication
+
+* External listeners (including `route`, `loadbalancer`, and `nodeport` types) for communication between Kafka clients and Kafka brokers
+
+Installing your own certificates and keys for plain listeners is not supported.
```

Comment:
I don't think this sentence is needed. Plain listeners aren't encrypted ... they are plain so TLS isn't in place, so having certificates don't make sense.

Reply:
Ah, I see. I'll remove this sentence.